### PR TITLE
feat(low-code): add new format float_s

### PR DESCRIPTION
**Pull request by tolik0**
## What
Add a new format - `%s_as_float` which represents time since the Unix epoch in seconds as a float, including microseconds. This format will be used in the Slack `channel_messages` stream.

## How
The **DatetimeParser** class is updated to handle the `%s_as_float` format for parsing and formatting datetime objects. This ensures that timestamps with microsecond precision are correctly interpreted and formatted.

## User Impact
Users will be able to accurately parse and format timestamps with microsecond precision. There are no negative side effects expected.

## Can this PR be safely reverted and rolled back?

- [ ] YES 💚
- [ ] NO ❌

### CODE REVIEW

**Review by girarda — COMMENTED** [review]
to better understand the rationale for the change: why is float a desired type? I'd expect an integer representing the number of microseconds to be easier to reason about that a float.

Specifically, in the context of Slack, is the Slack API returning datetimes as floats, or is that the result of how the connector is implemented?

**Comment by tolik0** [comment]
@girarda Yes, [the Slack API uses the float timestamp as the cursor](https://api.slack.com/methods/conversations.history#arg_oldest). Also, this is the format used by the `datetime` library:
```
    def timestamp(self):
        "Return POSIX timestamp as float"
        if self._tzinfo is None:
            s = self._mktime()
            return s + self.microsecond / 1e6
        else:
            return (self - _EPOCH).total_seconds()
```
I didn't find other connectors that use microseconds or float timestamps.

File: airbyte-cdk/python/airbyte_cdk/sources/declarative/datetime/datetime_parser.py [hunk 1]
```diff
@@ -27,6 +27,8 @@ def parse(self, date: Union[str, int], format: str) -> datetime.datetime:
         # See https://stackoverflow.com/a/4974930
         if format == "%s":
             return datetime.datetime.fromtimestamp(int(date), tz=datetime.timezone.utc)
+        elif format == "%float_s":
```
Comment (girarda) [inline airbyte-cdk/python/airbyte_cdk/sources/declarative/datetime/datetime_parser.py:30]: Let's call this `%s_as_float`

**Review by girarda — COMMENTED** [review]
Can you also update the doc in https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml#L720-L748
and 
https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml#L1718-L1744

File: airbyte-cdk/python/airbyte_cdk/sources/declarative/datetime/datetime_parser.py (hunk 1 above)
Comment (tolik0) [reply airbyte-cdk/python/airbyte_cdk/sources/declarative/datetime/datetime_parser.py]: Added documentation and updated the name

File: airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml [hunk 2]
```diff
@@ -1713,6 +1715,7 @@ definitions:
         description: |
           Format of the datetime value. Defaults to "%Y-%m-%dT%H:%M:%S.%f%z" if left empty. Use placeholders starting with "%" to describe the format the API is using. The following placeholders are available:
             * **%s**: Epoch unix timestamp - `1686218963`
+            * **%s_as_float**: Epoch unix timestamp in seconds as float - `1686218963.123456`
```
Comment (girarda) [inline airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml:1718]:
```suggestion
            * **%s_as_float**: Epoch unix timestamp in seconds as float with microsecond precision - `1686218963.123456`
```

File: airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml [hunk 3]
```diff
@@ -719,6 +719,7 @@ definitions:
         description: |
           The datetime format used to format the datetime values that are sent in outgoing requests to the API. Use placeholders starting with "%" to describe the format the API is using. The following placeholders are available:
             * **%s**: Epoch unix timestamp - `1686218963`
+            * **%s_as_float**: Epoch unix timestamp in seconds as float - `1686218963.123456`
```
Comment (girarda) [inline airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml:722]:
```suggestion
            * **%s_as_float**: Epoch unix timestamp in seconds as float with microsecond precision - `1686218963.123456`
```

**Review by girarda — APPROVED** [review]
small nit. lgtm!

**Pull request merged**